exchanges/kucoin/kucoin.go

```diff
@@ -1926,3 +1928,12 @@ func (ku *Kucoin) orderTypeToString(orderType order.Type) string {
 		return orderType.Lower()
 	}
 }
+
+func (ku *Kucoin) orderSideString(side order.Side) (string, error) {
+	switch side {
+	case order.Buy, order.Sell:
+		return strings.ToLower(side.String()), nil
+	default:
+		return "", errors.New("unsupported order side")
+	}
+}
```

Comment:
So a few things on how you've handled this

1. The function `SubmitOrder` will automatically verify if the order side is correct via the `Validate` function, the `AnySide` is not a valid side when submitting an order and will automatically be rejected.
2. You could use `side.Lower()` and minimise the amount of code here
3. As a result of point 1, you can add the case `order.AnySide` to return `""` instead of your err handling
4. This also means that you wouldn't need to update `GetOrderHistory` because that needed updating with your implementation